.github/workflows/generate_docs.yml

```diff
@@ -0,0 +1,40 @@
+name: Generate Documentation
+
+on:
+  push:
+    paths:
+      - pytermgui # run workflow only when files in the pytermgui directory changes
+    branches:
+      - master
+
+jobs:
+  generate_docs:
+    if: ${{ github.repository_owner == 'bczsalba' }} # prevents workflow from running in forked repos
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout master branch
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.x"
+
+      - name: Install pdoc
+        run: |
+          python -m pip install --upgrade pip
+          pip install pdoc
+
+      - name: Generate documentation
+        run: |
+          pdoc -o ./docs pytermgui
+
+      - name: Commit changes
+        run: |
+          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+          git config --local user.name "github-actions[bot]"
+          git add ./docs/
+          git commit -m "auto documentation update"
```

Comment:
The project's commit messages should be capitalized and uses an imperative style. Could you change it to `Auto-update documentation`?